src/Mikulas/OrmExt/StatefulProperty.php

```diff
@@ -32,12 +32,12 @@ public function __construct($state = NULL)
 		$this->stateMachine = new StateMachine($this);
 		$this->getLoader()->load($this->stateMachine);
 
+		$this->setFiniteState($state ?: $this->getInitialState());
+		$this->onModify();
```

Comment:
I am a little worried about this line - the callback might be using the stateMachine which has not yet been initialized. So maybe this call should be moved to the very end of the constructor? /cc @Mikulas